adr/cli.py

```diff
@@ -19,13 +19,21 @@
 log.addHandler(logging.StreamHandler())
 
 
+def print_to_file(data, filepath):
+    f = open(filepath, "x")
+    f.write(data)
+    f.close()
+
+
 def format_request(request, config, remainder, request_type):
 
     if request_type == "query":
         data, url = format_query(request, config, remainder)
     else:
         data = run_recipe(request, remainder, config)
         url = None
+    if config.output_file:
+        print_to_file(data, config.output_file)
```

Comment:
This is supported by the built-in `print` function, no need to create a helper:

```
print("foo", file=open(config.output_file, 'w'))
```

Reply:
Thanks I updated the code 👍